### PR TITLE
DDF-3889 updated docs template to publish architecture reference

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

adds to documentation templates to publish standalone architecture reference. 

#### Who is reviewing it? 
@mcalcote @ahoffer @garrettfreibott @austinsteffes 

#### Ask 2 committers to review/merge the PR and tag them here.
@lessarderic
@ricklarsen - Documentation
@rzwiefel
@shaundmorris

#### How should this be tested?

builds successfully

#### What are the relevant tickets?
[DDF-3889](https://codice.atlassian.net/browse/DDF-3889)

#### Checklist:
- [x] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -22,6 +22,7 @@ platform/solr/**/overlays
 distribution/docs/src/main/resources/content/_catalogFrameworks/.asciidoctor
 distribution/docs/src/main/resources/content/_catalogFrameworks/catalog_architecture_framework.png
 distribution/docs/src/main/resources/content/_catalogFrameworks/ingest_request.png	
+distribution/docs/src/main/resources/content/_catalogFrameworks/ingest_request.png
```
Comment (shaundmorris) [inline .gitignore]: ✏️  I might be missing it but lines 24 and 25 look the same.

Comment (ricklarsen) [reply .gitignore]: fixed

**Comment by ricklarsen** [comment]
build now

**Comment by ricklarsen** [comment]
build now

**Comment by ricklarsen** [comment]
build now

**Pull request merged**